src/panels/lovelace/cards/types.ts

```diff
@@ -80,6 +80,7 @@ export interface GaugeCardConfig extends LovelaceCardConfig {
 export interface ConfigEntity extends EntityConfig {
   tap_action?: ActionConfig;
   hold_action?: ActionConfig;
+  show_last_changed?: boolean;
```

Comment:
This should not be a generic option, this should only be added to the glance entity type.